packages/components/SimpleCard/index.js

```diff
@@ -26,18 +26,14 @@ class SimpleCard extends Component {
     handleEdit = () => {
       this.props.handleEdit(this.props.id);
     }
-
-    showLinkPath = () => {
-      return this.props.showLinkPath(this.props.id) || '';
-    }
-
+    
     render() {
 
       return (
         <Card style={ { width: '100%', marginBottom: '1em' } }>
           <Card.Body>
             <Row>
-              <LinkContainer to={ this.showLinkPath() } style={ { cursor: 'pointer' } }>
+              <LinkContainer to={ `${this.props.url}${this.props.id}` } style={ { cursor: 'pointer' } }>
```

Comment:
```suggestion
              <LinkContainer to={ `${this.props.url}` } style={ { cursor: 'pointer' } }>
```

I think at this point, the `url` should already have the `id` included, and the card doesn't need to know its `id`, which would imply somewhere at the parent level the `id` concatenation has happened.